src/index.js

```diff
@@ -0,0 +1,43 @@
+import { singleAnswerQuestion } from './utilities/singleAnswer';
+import { choiceQuestion } from './utilities/singleChoice';
+import { multiChoiceQuestion } from './utilities/multipleChoice';
+
+class Quiz {
+  constructor(questions) {
+    this.questions = questions;
+  }
+  start() {
+    let result = 0;
+    this.questions.forEach(function (question) {
+      const userAnswer = window.prompt(question.formatQuestion());
+      const formattedUserAnswer = userAnswer.toLowerCase().replace(/\s/g, '');
+      if (formattedUserAnswer === question.answer) {
+        result++;
+        console.log('Correct!');
+      } else {
+        console.log('Wrong!');
+      }
+    });
+    console.log(`The user got ${result}/${this.questions.length} points`);
+  }
+}
+
+const quiz = new Quiz([
```

Comment:
You're missing the short answer question.

![image](https://github.com/user-attachments/assets/99edab73-9a0a-4ecf-8334-fc38039c1d3c)